### DISCUSSION TITLE
CC:Tweaked 1.113.0 missing on maven

### DISCUSSION THREAD

**Discussion by dimaguy** [cc-tweaked/CC-Tweaked#1965] [Q&A] [answered]
Hey there, I've managed to make it so Plethora-Fabric works with CC:Tweaked 1.113.0 (1.20.1), but I've had to use Maven Local to do so, which isn't very practical, could that version be published on the regular maven place?

**Comment by SquidDev** [comment, marked as answer]
CC:T's maven has changed. You'll need to update it to `https://maven.squiddev.cc` instead. See [the README](https://github.com/cc-tweaked/CC-Tweaked?tab=readme-ov-file#using) for more info.

**Reply by dimaguy** [reply]
Oops, sorry for bothering then! I hadn't noticed :/
